app/PIOCIE/PIOCIE_task_3.c

```diff
@@ -0,0 +1,85 @@
+/**
+ * Responsible: Piotr Ciesielczyk
+ *
+ *
+ * History log:
+ * 2017.12.07   PioCie  Initial version that complete tests
+ */
+
+#include <stdio.h>
+#include "PIOCIE_task_3.h"
+#define uint8_t unsigned char
+/* Every public function in your module should start with   "MODULENAME_"   prefix */
+/* Every private function in your module should start with  "MODULENAME__"  prefix */
+
+static uint8_t g_u8rgStorage[LAMP_COUNT%8?LAMP_COUNT/8+1:LAMP_COUNT/8];
+
+eErr_t PIOCIE_lampOn(eLamp_t eLamp)
+{
+	eErr_t result;
```

Comment:
Task 3 - Use hungarian notation (http://web.mst.edu/~cpp/common/hungarian.html). Search the internet for more examples. This is mandatory for task acceptance  (here eResult)